bamboo/deploy-dev-integration-test-stack.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -ex
+. ./bamboo/abort-if-not-pr-or-redeployment.sh
+. ./bamboo/abort-if-skip-integration-tests.sh
+. ./bamboo/set-bamboo-env-variables.sh
+
+npm config set unsafe-perm true
+
+if [[ $USE_TERRAFORM_ZIPS == true ]]; then
+  ## If this flag is set, we want to use the output of the 'publish' stage
+  ## to deploy in the integration test stage, so abort
+  exit 0;
+fi
+
+if [[ $USE_CACHED_BOOTSTRAP == true ]]; then
+  ## Change into cached cumulus, pull down /cumulus ref and run there
+  echo "*** Using cached bootstrap"
+  cd /cumulus/
+  git fetch --all
+  git checkout "$GIT_SHA"
+fi
+echo "***Deploying stack with built source"
+
+npm install
+## Double bootstrapping required as workaround to
+## lerna re-bootstrapping issue in older releases
+## (similiar to  https://github.com/lerna/lerna/issues/1457)
+npm run bootstrap-no-build && npm run bootstrap-no-build
```

Comment:
Obviously I'm confused, but I don't get why the bootstrap commands are run multiple times in these scripts if the proposed improvement is that the result of this operation is cached

Reply:
Not the *result* of the operation.  The bootstrapping in the scripts is updating the cache to match the current state of the checked out code.